Makefile

```diff
@@ -37,13 +37,16 @@ LXC_BRANCH?=stable-5.0
 HACK_D := $(TOP_LEVEL)/hack
 # helper tools
 TOOLS_D := $(HACK_D)/tools
+export PATH := $(TOOLS_D)/bin:$(PATH)
```

Comment:
Does this even work with sudo -E?

Reply:
```
$ export PATH=/tmp/foo:$PATH ; sudo -E sh -c 'echo $PATH'
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin
```
[sudo(8)](https://manpages.ubuntu.com/manpages/jammy/en/man8/sudo.8.html) says:

>  PATH             May be overridden by the security policy.

So i guess this doesn't work at least in the default ubuntu path here.

So I guess we have to leave PATH=PATH.